Dev-Corner/slides/tool_integration.html

```diff
@@ -59,103 +59,1315 @@
+
+                            #### The Goal
+                            At functional test to the Seqtk wrapper for Seqtk
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### <i class="fa fa-pencil" aria-hidden="true"></i> Hands-on
+
+                            #### Steps
+                            1. Fill a `<test>` tag using the file you generate previously with `seqtk`
+                            2. Use `planemo test --install_galaxy ...` until all is green
+
+                        </script>
+                    </section>
+                </section>
+
+                <!-- DEPENDENCIES -->
+                <section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ## Dependencies
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Dependencies
+                            #### How Galaxy will deal with dependencies?
+
+                            <img src="../images/galaxy_instance.png" alt="Galaxy instance" height=500/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Dependencies
+                            #### Embedded tool
+
+                            ```
+                            graphlan/
+                            ├── graphlan.xml
+                            ├── test_data/
+                                ├── input_tree.txt
+                                ├── png_image.png
+                            ├── graphlan.py
+
+                            ```
+
+                            ```
+                            <command><![CDATA[
+                                $__tool_directory__/graphlan.py
+                                    ...
+                            ]]></command>
+                            ```
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Dependencies
+                            #### External tool
+
+                            Definition of requirements in `.xml` file
+
+                            ```
+                            <requirements>
+                                <requirement type="package" version="1.66">biopython</requirement>
+                                <requirement type="package" version="1.0.0">graphlan</requirement>
+                            </requirements>
+                            ```
+
+                            Local installation using ~~tool_dependencies.xml~~
```

Comment:
Unfortanatly, all the tools are not available under conda packages.
They will meet this term in docs, via the list galaxydev ...
Furthermore some already know this mecanism.